[docs] Reorganize hugo doc site

anik120 opened the pull request:
**Description of the change:**

This PR reorganizes sdk.operatorframework.io to match the navigation
panel for olm.operatorframework.io, so that both the sites appear
to be part of one operator-framework family.

**Motivation for the change:**


**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

jmrodri commented:
@camilamacedo86 I found it easier to review by looking at the preview site:
* Preview of this PR: https://deploy-preview-3382--operator-sdk.netlify.app/docs/
* Existing SDK docs site: https://sdk.operatorframework.io/docs/

Quite a bit of changes there.

jmrodri commented:
Heading font size is much smaller than before and doesn't seem to match the one used on OLM either. 
The font from this PR: 
![image](https://user-images.githubusercontent.com/17137/87190584-b9817580-c2c0-11ea-99f1-c6cb4b8348b8.png)

The old SDK fonts were much larger:
![image](https://user-images.githubusercontent.com/17137/87190616-c605ce00-c2c0-11ea-9e9e-2408436f3016.png)

The OLM fonts seem to be slightly larger compared to the one used in this PR:
![image](https://user-images.githubusercontent.com/17137/87190664-de75e880-c2c0-11ea-8a63-be280ee065c0.png)

jmrodri commented:
Core Tasks seems odd to me. 
![image](https://user-images.githubusercontent.com/17137/87190931-6fe55a80-c2c1-11ea-8353-7253c7105cea.png)

It seems to fit on the OLM side because the steps seem the same. But the SDK has the different types of operators which have their own audiences and presumably a different set of "core tasks".

jmrodri commented:
Also, I think `Getting Started` should be called `Installation` on both sites since both of the docs just describe how to install SDK or OLM:
* Getting started from PR: https://deploy-preview-3382--operator-sdk.netlify.app/docs/getting-started-/
* OLM Getting Started: https://olm.operatorframework.io/docs/getting-started/
* Installation from current SDK site: https://sdk.operatorframework.io/docs/install-operator-sdk/

jmrodri commented:
I think I know why `Core Tasks` and `Advanced Tasks` seem odd. I think it should be `Core Topics` and `Advanced Topics` because on the SDK side they really aren't "tasks" per se.

camilamacedo86 commented on website/content/en/docs/getting-started /_index.md:
```diff
@@ -1,7 +1,10 @@
-title: Install the Operator SDK CLI
-linkTitle: Installation
-weight: 1
+title: "Getting Started"
```
I agree with @jmrodri. 
The contact here is about how to install so IMO it should be `Installation`
The Getting Started content is in https://github.com/operator-framework/getting-started which I think we should add here. IMO we could just copy and paste now and then, do the follow-ups required to fix it as we wish or todo it in another PR. @dmesser @jmrodri   wdyt?

camilamacedo86 commented on website/content/en/docs/core-tasks/bumping-sdk-version/_index.md:
```diff
@@ -0,0 +1,5 @@
+---
+title: Operator SDK Version migration
```
It is inside of the "core task" which I also think that needs to be renamed. 
1. However, IMO it should be outside on top of Contribution Guidelines. WDYT? 

Also, find a better name to illustrate its purpose would be nice as well. 
2. I'd suggest replacing `Operator SDK Version migration` for `Upgrade SDK Version`. See;

<img width="257" alt="Screenshot 2020-07-10 at 20 59 55" src="https://user-images.githubusercontent.com/7708031/87193321-60313a80-c2f0-11ea-81d0-a6c3d65b5827.png">

3. Do we really need keep these `Migration` doc from  v0.0.x to v0.1.0?

anik120 replied:
@camilamacedo86 at first glance of the docs it feels like the ordering should be more like: 

- Operator SDK Version upgrade guide
- v0.0.x to v0.1.0
- v0.0.x to v0.18.0
- v0.0.x to v0.19.0

I'm updating the PR, please let me know if I misread the docs/it needs more reordering.

camilamacedo86 commented on website/content/en/docs/core-tasks/bumping-sdk-version/version-upgrade-guide.md:
```diff
@@ -2,10 +2,9 @@
 title: Operator SDK Version upgrade guide
 linkTitle: Version Upgrade
```
```suggestion
linkTitle: 0.1.0 to 0.17.x
```

camilamacedo86 replied:
```suggestion
linkTitle: 0.1.0 to 0.17.x
```

camilamacedo86 commented on website/content/en/docs/core-tasks/bumping-sdk-version/version-upgrade-guide.md:
```diff
@@ -2,10 +2,9 @@
 title: Operator SDK Version upgrade guide
```
```suggestion
title: Operator SDK Version upgrade guide (0.1.0 to 0.17.x)
```

camilamacedo86 replied:
```suggestion
title: Operator SDK Version upgrade guide (0.1.0 to 0.17.x)
```

camilamacedo86 commented on website/content/en/docs/core-tasks/bumping-sdk-version/v0.1.0-migration-guide.md:
```diff
@@ -1,11 +1,10 @@
 ---
 title: Operator SDK Migration Guide from v0.0.x to v0.1.0
 linkTitle: Migration
-weight: 1
+weight: 2
+description: A guide to migrate an operator project built using Operator SDK `v0.0.x` to the project structure required by `v0.1.0`.
 ---
```
@jmrodri @hasbro17 @asmacdo @fabianvf do we really need to keep this doc?

camilamacedo86 commented on website/content/en/docs/core-tasks/bumping-sdk-version/v0.1.0-migration-guide.md:
```diff
@@ -1,11 +1,10 @@
 ---
 title: Operator SDK Migration Guide from v0.0.x to v0.1.0
 linkTitle: Migration
```
```suggestion
linkTitle: v0.0.x to v0.1.0
```

camilamacedo86 commented on website/content/en/docs/core-tasks/_index.md:
```diff
@@ -0,0 +1,9 @@
+
+---
+title: "Core Tasks"
+linkTitle: "Core Tasks"
+weight: 3
```
WDYT? 

```suggestion
title: "Operator SDK user guides by Operator Types"
linkTitle: "Operator Types"
weight: 3
```

camilamacedo86 replied:
@jmrodri @jmccormick2001 @fabianvf @asmacdo

camilamacedo86 replied:
Also, see that the submenus are very prolix/redundant : 

<img width="309" alt="Screenshot 2020-07-10 at 21 12 02" src="https://user-images.githubusercontent.com/7708031/87197338-409b1180-c2f2-11ea-9a8a-b219103305fe.png">


I would recommend just Ansible, Golang and Helm which would be 

- Documentation
|
----- Operator Types
|
------------ Ansible
|
------------ Helm
|
------------ Golang

tengqm replied:
Or the `linkTitle` can be just `"Building Operators"`?

anik120 replied:
@tengqm's idea seems like it'll drive home the point about what the section is for in first glance. Updating the PR with his suggestion for the `linkTitle` and removing redundancies @camilamacedo86 mentioned.

camilamacedo86 commented:
The link in https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/main.go#L51 also needs to be updated.

anik120 commented:
Thanks a lot for review on the PR @camilamacedo86 @jmrodri

I have a few follow up questions/comments: 

@jmrodri 
> Also, I think Getting Started should be called Installation on both sites since both of the docs just describe how to install SDK or OLM 

OLM getting started is about  1) installing OLM, and then 2) Installing an operator using OLM. So it's a very basic, "Quickstart guide" to using OLM, which is in other words: "Get started with OLM" 

However I understand your point about SDK's getting started being about just installing SDK. I'd named that section "Getting started" to be consistent with OLM, but if you're really against that then I don't mind renaming the SDK "Getting started" to "Installing SDK" 

@camilamacedo86 could you help me understand the broken links error? From the report I can see: 

```ERROR: 4 dead links found!
[✖] ./website/content/en/docs/scorecard/_index.md#config-file → Status: 400
[✖] ./website/content/en/docs/cli/operator-sdk_cleanup.md → Status: 400
[✖] ./website/content/en/docs/cli/operator-sdk_run.md → Status: 400
[✖] ./website/content/en/docs/scorecard/_index.md → Status: 400
```
However, the "See the `config file` " link [here](https://sdk.operatorframework.io/docs/scorecard/scorecard-alpha/#running-the-scorecard) seems like it isn't broken. 

Maybe I'm reading the error report wrong?

anik120 commented:
@camilamacedo86 @jmrodri addressed your comments. PTAL and lmk what you think, thank! 

ps: The comment about font size still needs to be addressed, trying to assess what effort that would entail. Will update here as soon as I have something.

camilamacedo86 commented:
Hi @anik120, 

Note that it is not passing in the tests.  Following some comments and my review. 

**Regards the CI**

1. You changed the CLI docs, however, they are generated automatically via the Makefile target `make generate`. So, you also need to change its scripts. See the CI issue in https://travis-ci.org/github/operator-framework/operator-sdk/jobs/708135863 and its script in `./hack/generate/cli-doc/gen-cli-doc.sh`

**Regards the section Building Operators**

2. In the menu we have now: `Building Operators` >> `Golang` >> `Legacy CLI and Project Layout`. Could we replace it just for Legacy? See that big names are broked and we will find in this section just the Legacy docs but not the CLI ones(these are generated automatically). See:

<img width="838" alt="Screenshot 2020-07-15 at 11 36 19" src="https://user-images.githubusercontent.com/7708031/87535445-6c6e1c80-c68f-11ea-8ca9-3929fccdbf1f.png">

3. In the above image, we can see `Golang Based Operator Quickstart (Legacy)` inside of the `Legacy` submenu. However, the other docs have not the `(Legacy)` annotation. So could we add it in legacy docs? WDYT?

**Regards Upgrade SDK Version**

<img width="253" alt="Screenshot 2020-07-15 at 11 40 14" src="https://user-images.githubusercontent.com/7708031/87536129-80fee480-c690-11ea-8986-7ec504fed4de.png">

4. `Upgrade SDK Version` is inside of `Building Operators`. IMO it should be at the same level of `Building Operators` and not inside of this submenu. 

5. The `Version Upgrade` submenu should be replaced/renamed to `v0.2.x to v0.17.x` to make clear that it has the steps from  `v0.2.x to v0.17.x` version.  
 
6. You add invalid information to the migration guide `v0.18.x`. It is now [here](https://deploy-preview-3382--operator-sdk.netlify.app/docs/building-operators/upgrading-sdk-version/v0.18.0/) and the original is [here](https://sdk.operatorframework.io/docs/migration/v0.18.0/)
- Its menu should NOT be `v0.0.x to v0.18.0` and instead it should be `v0.18.0` as it is today or `v0.17.x to v0.18.0`
- Could you please, remove the info added on it `A guide to migrating an operator project built using Operator SDK v0.0.x to the project structure required by v0.18.0` and keep it as the original. It is NOT accurate. 

7. You add invalid information to the migration guide `v0.19.x`. It is now [here](https://deploy-preview-3382--operator-sdk.netlify.app/docs/building-operators/upgrading-sdk-version/v0.19.0/) and the original is [here](https://sdk.operatorframework.io/docs/migration/v0.19.0/)
- Its menu should NOT be `v0.0.x to v0.19.0` and instead it should be `v0.19.0` as it is today or `v0.18.x to v0.19.x`
- Could you please, remove the info added on it `A guide to migrating an operator project built using Operator SDK v0.0.x to the project structure required by v0.19.0` and keep it as the original. It is NOT accurate. 

8. Would not the doc `Versioning` be part of this submenu? See https://deploy-preview-3382--operator-sdk.netlify.app/docs/concepts/operator-sdk-versioning/. It explains how the SDK is versioned so I think would fit better here. 

9. Would not the doc `Backport Policy` be part of this submenu? See https://deploy-preview-3382--operator-sdk.netlify.app/docs/contribution-guidelines/backport-policy/ it explains how the patch releases work and when they are made. 

10. Could we please perform the above fixes and keep the submenu sorted as: 

Inside of `Upgrade SDK Version`:

> - Versioning
> - Backport Policy
>- v0.0.x to v0.1.0
>- v0.2.x to v0.17.x
>- v0.17.x to v0.18.x 
>- v0.18.x to v0.19.x 

**Regards `Concepts`**

<img width="236" alt="Screenshot 2020-07-15 at 12 13 55" src="https://user-images.githubusercontent.com/7708031/87538750-ac83ce00-c694-11ea-956a-b9c2bdbc1691.png">

11. Note that following these suggestions inside of `Concepts` we will have just `Operator Capability Levels`. So, why not keep it inside of the `Advanced Topics`? And see that the `Concepts` has just 2 docs currently.

**Regards Operator SDK CLI**

<img width="197" alt="Screenshot 2020-07-15 at 12 14 52" src="https://user-images.githubusercontent.com/7708031/87538833-cfae7d80-c694-11ea-9355-8ccd7d18fbd1.png">

12. Do we really use the word `Operator` again? Could we not just keep `Commands (CLI)` instead or a better name that reflects that it is just the docs over the SDK tool commands (CLI)? 

13. Then, we have `Ansible/Helm CLI Reference` and `Golang CLI Reference`. IMO: The `CLI Reference` is prolix/verbose/repetitive so WDYT about keep just  `Ansible/Helm` and `Golang` as it is in the `Building Operators`?

**Regards OLM Integration**

<img width="240" alt="Screenshot 2020-07-15 at 12 18 31" src="https://user-images.githubusercontent.com/7708031/87539166-56fbf100-c695-11ea-8438-8e384e8fbd47.png">

14. See item 3 here. IMO we need to keep the same standard. So, WDYT about replace `Legacy Docs` for `Legacy` and then, update its internal docs with `(Legacy)` to make clear that they are legacy ones?  

**Migration Guide link in the Deprecated Notice**

15. You will need also update the link that appears for the users to point out to migration guide. See: https://github.com/operator-framework/operator-sdk/pull/3395

Really thank you for your contribution. It is terrific.  In POV, the docs after that still requiring follow-ups for we better organize its contents and etc but shows that started to organize them in the menu it is a great start.

anik120 commented:
@camilamacedo86 could you take a look at it now. 

IMHO the `legacy` stuff is really confusing, aren't we supposed to put the legacy stuff in previous versions of the site now that we have versioned docs? 
But that also could be because I'm looking at these with bird's eye view so maybe I'm wrong. 

Also, I think travis is still failing, not sure what I'm doing wrong though.

camilamacedo86 commented:
Hi @anik120, 

I just discovered that we might be able to remove the legacy for 1.0.0. So, ignore my suggestion over add `(legacy)`. I think is not part of the scope of this task remove them. we need to see if something would still be required. However, I think it also solves your concerns. 👍

camilamacedo86 commented:
Hi @anik120,

If you see in the master we already removed many of them. So, feel free to remove the legacy here as well.

anik120 commented:
@camilamacedo86 if it's okay I'd like to get this merged now. I've rebased off the latest master, and it should be in a state now to incrementally build on top of (remove any remaining legacy etc).

anik120 commented:
@camilamacedo86 thank you! It should be fixed now. 

Maybe @jmrodri can /approve it if you're satisfied with it? 
PS: About the font size, I didn't actually change any of that myself in this PR, but I'm guessing it's coz the setup wasn't done right for the UI. We can open up a follow up PR to fix the font size etc if that's okay.

jmrodri commented:
/lgtm
/approve

jmrodri commented:
@anik120 I'm okay with fixing the fonts at a later time.

camilamacedo86 commented:
Hi @anik120, 

It still not passing in the CI. We cannot merge without that. 

**Regards the sanity check**

- You changed the CLI docs, however, they are generated automatically via the Makefile target make generate. So, you also need to change its scripts. See the CI issue in https://travis-ci.org/github/operator-framework/operator-sdk/jobs/708135863 and its script in ./hack/generate/cli-doc/gen-cli-doc.sh
- You also need to run `make generate` locally. Then, the docs should be generated in the new path. 
- You can test it locally by running `make test-sanity` (note that all changes need to be committed because it will do a git diff at the end)

**Regards the docs**
It has a lot of broken links now. You need to fix the links ref in the docs. See: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/708825906

anik120 commented:
@camilamacedo86 could you take a look now please. I've fixed most of it, there's a lot of external links that are broken. Not sure if I have to do anything about that in this PR.

jmrodri commented:
tons of broken links because of all the churn happening in master with legacy being removed.
```
 *  internally linking to docs/building-operators/golang/legacy/e2e-tests, which does not exist (line 1368)

     <a href="docs/building-operators/golang/legacy/e2e-tests">Writing E2E Tests</a>

- /target/docs/building-operators/golang/legacy/e2e-tests/index.html

  *  External link https://github.com/operator-framework/operator-sdk/blob/master/pkg/test/context.go failed: 404 No error

  *  External link https://github.com/operator-framework/operator-sdk/blob/master/pkg/test/framework.go failed: 404 No error

  *  External link https://github.com/operator-framework/operator-sdk/tree/master/pkg/test/e2eutil failed: 404 No error

  *  internally linking to /docs/cli/legacy-cli/operator-sdk_test_local, which does not exist (line 1562)

     <a href="/docs/cli/legacy-cli/operator-sdk_test_local">SDK CLI Reference</a>

- /target/docs/building-operators/golang/quickstart/index.html

  *  External link https://v0-19-x.sdk.operatorframework.io/docs/building-operators/golang/legacy/quickstart/ failed: 404 No error

- /target/docs/building-operators/helm/reference/watches/index.html

  *  internally linking to /docs/helm/building-operators/reference/advanced_features/#passing-environment-variables-to-the-helm-chart, which does not exist (line 1363)

     <a href="/docs/helm/building-operators/reference/advanced_features/#passing-environment-variables-to-the-helm-chart">Using override values and passing environment variables to the Helm chart</a>

  *  linking to internal hash #passing-environment-variables-to-the-helm-chart that does not exist (line 1363)

     <a href="/docs/helm/building-operators/reference/advanced_features/#passing-environment-variables-to-the-helm-chart">Using override values and passing environment variables to the Helm chart</a>

- /target/docs/cli/operator-sdk_scorecard/index.html

  *  internally linking to ../operator-sdk, which does not exist (line 1376)

     <a href="../operator-sdk">operator-sdk</a>

- /target/docs/contribution-guidelines/release/index.html

  *  internally linking to /docs/upgrade-sdk-version/backport-policy, which does not exist (line 1400)

     <a href="/docs/upgrade-sdk-version/backport-policy">backport policy</a>

  *  internally linking to docs/installation/install-operator-sdk#prerequisites, which does not exist (line 1365)

     <a href="docs/installation/install-operator-sdk#prerequisites">prerequisites section</a>

  *  internally linking to docs/installation/install-operator-sdk#prerequisites, which does not exist (line 1367)

     <a href="docs/installation/install-operator-sdk#prerequisites">prerequisites section</a>

  *  internally linking to docs/installation/install-operator-sdk, which does not exist (line 1489)

     <a href="docs/installation/install-operator-sdk">installation guide.</a>

  *  linking to internal hash #prerequisites that does not exist (line 1365)

     <a href="docs/installation/install-operator-sdk#prerequisites">prerequisites section</a>

  *  linking to internal hash #prerequisites that does not exist (line 1367)

     <a href="docs/installation/install-operator-sdk#prerequisites">prerequisites section</a>

- /target/docs/new-cli/operator-sdk_scorecard/index.html

  *  internally linking to ../operator-sdk, which does not exist (line 1378)

     <a href="../operator-sdk">operator-sdk</a>

- /target/docs/olm-integration/generating-a-csv/index.html

  *  internally linking to /docs/advanced-topics/operator-capabilities/, which does not exist (line 1528)

     <a href="/docs/advanced-topics/operator-capabilities/">Operator maturity model</a>

- /target/docs/overview/_overview/index.html

  *  internally linking to /docs/advanced-topics/operator-capabilities/, which does not exist (line 1391)

     <a href="/docs/advanced-topics/operator-capabilities/">capability level documentation</a>

  *  internally linking to /docs/faq/, which does not exist (line 1395)

     <a href="/docs/faq/">FAQ</a>

- /target/docs/upgrading-sdk-version/v0.1.0-migration-guide/index.html

  *  External link https://v0-19-x.sdk.operatorframework.io/docs/building-operators/golang/legacy/quickstart/#adding-3rd-party-resources-to-your-operator failed: 404 No error

  *  External link https://v0-19-x.sdk.operatorframework.io/docs/building-operators/golang/legacy/quickstart/#build-and-run-the-operator failed: 404 No error

  *  External link https://v0-19-x.sdk.operatorframework.io/docs/building-operators/golang/references/client/ failed: 404 No error
```

jmrodri commented:
And now there is a conflict so this PR needs a rebase

anik120 commented:
@camilamacedo86 @jmrodri resolved all the broken links expect for [these two](https://travis-ci.org/github/operator-framework/operator-sdk/jobs/709232668#L322) and looks like they're broken in [main branch](https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/new-cli/operator-sdk_scorecard.md#see-also) too.

camilamacedo86 commented on CHANGELOG.md:
```diff
@@ -220,7 +220,7 @@
 - The type name `TestCtx` in `pkg/test` has been deprecated and renamed to `Context`. It now exists only as a type alias to maintain backwards compatibility. Users of the e2e framework should migrate to use the new name, `Context`. The `TestCtx` alias will be removed in a future version. ([2549](https://github.com/operator-framework/operator-sdk/pull/2549))
```
Revert the changes in the Change in the changelog

camilamacedo86 commented:
Hi @anik120, 

See that it still with broken links:

error msg: 

```
- /target/docs/cli/operator-sdk_scorecard/index.html
  *  internally linking to ../operator-sdk, which does not exist (line 1194)
     <a href="../operator-sdk">operator-sdk</a>
```

It means that the `../operator-sdk` in `/docs/cli/operator-sdk_scorecard.md` do not exist. See: 

https://github.com/operator-framework/operator-sdk/blob/2b366c4c969d2647fa61ea7b279f9c67e12e60fb/website/content/en/docs/cli/operator-sdk_scorecard.md 

same for: 

```
- /target/docs/new-cli/operator-sdk_scorecard/index.html
  *  internally linking to ../operator-sdk, which does not exist (line 1196)
     <a href="../operator-sdk">operator-sdk</a>
```

Then, see that it was changed in the PR https://github.com/operator-framework/operator-sdk/pull/3444 and we have no longer where we have no longer 2 versions of the tool. The alpha(new cli) is the only one now. 

Also, see that something goes wrong in your rebase: 

<img width="260" alt="Screenshot 2020-07-17 at 18 31 42" src="https://user-images.githubusercontent.com/7708031/87814507-c91e3280-c85b-11ea-9e93-28d080252378.png">

The scorecard should be a many with its docs instead that. Also, see that the its cli index doc is also in the root when should not. See the [master](https://master.sdk.operatorframework.io/docs/scorecard/) branch docs to check how it should be: 

<img width="233" alt="Screenshot 2020-07-17 at 18 33 42" src="https://user-images.githubusercontent.com/7708031/87814674-100c2800-c85c-11ea-8e13-24754d07511c.png">

camilamacedo86 commented on hack/generate/cli-doc/gen-cli-doc.go:
```diff
@@ -39,12 +39,12 @@ func main() {
 
-	newDocPath := filepath.Join(currentDir, "website", "content", "en", "docs", "new-cli")
+	newDocPath := filepath.Join(currentDir, "website", "content", "en", "docs", "cli", "new-cli")
 	_, newRoot := cli.GetPluginsCLIAndRoot()
```
revert these changes

estroz commented on website/content/en/docs/cli/ansible-helm/_index.md:
```diff
@@ -0,0 +1,18 @@
+
+**Note:** For Golang Operators this CLI has been deprecated. Please consult the [new CLI reference][new_CLI] doc.
+
+[new_CLI]:/docs/cli/new-cli
```
Incorrect link:
```suggestion
[new_CLI]:/docs/new-cli
```

We might want to wait until #3455 is merged to simplify this PR

estroz replied:
Never mind, we can merge #3455 after this one is merged.

anik120 replied:
@estroz yea that'd be appreciated this PR has been in the works for a week now. 
Although, CI job request is still pending for this one.

estroz commented on website/content/en/docs/cli/golang/_index.md:
```diff
@@ -0,0 +1,9 @@
+
+**Note:** This CLI reference is for the new Kubebuilder aligned CLI and project layout. See the [legacy CLI reference][legacy_cli] doc if using a legacy Golang project.
+
+[legacy_CLI]:/docs/cli/legacy-cli
```
```suggestion
[legacy_CLI]:/docs/cli
```

jmrodri commented:
/lgtm
